graphistry/feature_utils.py

```diff
@@ -2187,14 +2335,17 @@ def scale(
                     encode=encode,
                     strategy=strategy,
                     keep_n_decimals=keep_n_decimals,
+                    return_pipeline=True
                 )  # type: ignore
             else:
                 raise AttributeError(
                     'Please run g.featurize(kind="edges", *args, **kwargs) '
                     'first before scaling matrices and targets is possible.'
                 )
+        if return_scalers:
+            return X, y, scaler, scaler_target
+        return X, y
```

Comment:
return X, y, None, None would have been nicer for unit tests,

Reply:
But messy elsewhere -- unless user wants those scaling pipelines for downstream modeling, which they don't really need since they have the graphistry instance, I decided to keep `X_scaled, y_scaled` the default behavior.